go.mod

```diff
@@ -1,3 +1,10 @@
 module go.opentelemetry.io/proto
 
 go 1.15
+
+require (
+	github.com/grpc-ecosystem/grpc-gateway/v2 v2.10.3
+	go.opentelemetry.io/proto/otlp v0.16.0
+	google.golang.org/grpc v1.47.0
+	google.golang.org/protobuf v1.28.0
+)
```

Comment:
These look like side-effects from the build, I'm not able to reproduce them locally when duplicating the build.

```suggestion
```

Reply:
Yeah, the tool told me I had to run "go mod tidy" and this happened.